keps/sig-cloud-provider/providers/0004-cloud-provider-template.md

```diff
@@ -0,0 +1,90 @@
+---
+kep-number: 4
+title: Cloud Provider Template
+authors:
+  - "@janedoe"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-aaa
+  - sig-bbb
+reviewers:
+  - TBD
+  - "@alicedoe"
+approvers:
+  - "@andrewsykim"
+  - "@hogepodge"
+  - "@jagosan"
+editor: TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional
+see-also:
+  - KEP-1
+  - KEP-2
+replaces:
+  - KEP-3
+superseded-by:
+  - KEP-100
+---
+
+# Cloud Provider FooBar
+
+This is a KEP template, outlining how to propose a new cloud provider into the Kubernetes ecosystem.
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Requirements](#requirements)
+* [Proposal](#proposal)
+
+## Summary
+
+This is where you add a summary of your cloud provider and other additional information about your cloud provider that others may find useful.
+
+## Motivation
+
+### Goals
+
+This is where you can specify any goals you may have for your cloud provider.
+
+### Non-Goals
+
+This is where you can specify any work that you think is outside the scope of your cloud provider.
+
+## Prerequisites
+
+This is where you outline all the prerequisites for new providers that have been met.
+
+### Repository Requirements
+
+For [repository requirements](https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0002-cloud-controller-manager.md#repository-requirements) you are expected to have a repo (belonging to any organization, ideally owned by your cloud provider) that has a working implementation of the [Kubernetes Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/). Note that the list of requirements may be changing in the near future.
+
+### User Experience Reports
+
+There must be a reasonable amount of user experience report for running Kubernetes for this cloud provider. You may want to link to sources that indicate this such as github issues, product data, customer tesitimonials, etc.
+
+
+## Proposal
+
+This is where you can talk about what resources from the Kubernetes community you would like such as a repository in the Kubernetes organization to host your provider code.
+
+### Subproject Leads
+
+This is where you indicate any leads for the subproject. Make sure you include their github handles. See the [SIG Charter](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/CHARTER.md#subprojectprovider-owners) for more details on expectations from subproject leads.
+
+### Repositories
```

Comment:
probably should also note that a PR to the SIG Cloud Provider subproject list will also be required for all repositories to create/transfer